src/verilog/verilog_expr.h

```diff
@@ -14,9 +14,14 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/expr.h>
 
 /// The syntax for these A.B, where A is a module identifier and B
-/// is an identifier within that module.
+/// is an identifier within that module, given als symbol_exprt.
```

Comment:
I think this needs some editing. How about: "The syntax of hierarchical Verilog identifiers is A.B, where A is a module identifier B refers to a symbol within that module. B is given as symbol_exprt."